rs/execution_environment/benches/wasm_instructions/basic.rs

```diff
@@ -30,21 +40,21 @@ pub fn benchmarks() -> Vec<Benchmark> {
         let ty = dst_type(op);
         let name = format!("const/{op}");
         let code = &format!("(global.set $x_{ty} ({op} 7))");
-        benchmarks.extend(benchmark_with_confirmation(&name, code));
+        benchmarks.extend(benchmark_with_confirmation(&name, code, wasm64_enabled));
```

Comment:
Would it be easier to just modify the `benchmark_with_confirmation` function to return benchmarks for Wasm32 and Wasm64?

Reply:
Not sure I understand -- does that mean that in "one go" we run all benchmarks in both Wasm32 or Wasm64?

Reply:
What seems to be the problem?

Reply:
How would you then differentiate between Wasm64 and Wasm32 modes when a result comes in? Would you like to add an extra tag to each instruction, something like `wasm32/memop/i32.load`?